### PR TITLE
637:  Remove school subdistrict enrollment multiplier table & logic

### PR DESCRIPTION
**Pull request by bfreeds**
Previously this table of percentages was brought into the rails application to be used in the multipliers to calculate projected enrollment. With EDM taking ownership over the data pipelines, they will include this calculation in their pipeline, giving us a table that's precalculated.  

Read more about context of this process in issue #637 

**Note** we use `ceqr_data` db for development, staging, and production.  So we don't want to remove this table from the db until all the logic in the app has already been updated & the new data is fully implemented.

Addresses #637

### CODE REVIEW

**Comment by trbmcginnis** [comment]
I see that you removed tests for lcgms and bluebook datasets. 

 I already did this in the ceqr_school_buildings data update PR #635 and I don't think that should be handled in this PR since it's not related to `sca_enrollment_pct_by_sd` data changes

File: backend/spec/models/public_schools_analysis_spec.rb [hunk 1]
```diff
@@ -58,20 +58,11 @@
...
-  end
-
   it "sets hs_projections correctly" do
     borough = project.public_schools_analysis.hs_projections.map {|n| n['borough']}
     expect(borough).to eq(['brooklyn'])
   end
 
-  it "sets future_enrollment_projections correctly" do
-    expect(project.public_schools_analysis.future_enrollment_projections[0]['district']).to eq(15)
-  end
```
Comment (trbmcginnis) [inline backend/spec/models/public_schools_analysis_spec.rb]: I don't think you should remove this test. We are not removing `future_enrollment_projections` from the model, we are only removing `future_enrollment_multipliers`.

Comment (bfreeds) [reply backend/spec/models/public_schools_analysis_spec.rb]: You're totally right - thanks for catching, fixed this in ee9e318103155f3d21232f63be0f37f3f78a9c71

**Comment by bfreeds** [comment]
@trbmcginnis Good catch on the accidentally removed datasets, which are already covered in your PR #635.  I removed that commit from this PR.

**Comment by trbmcginnis** [comment]
Closing because this is now covered in PR #642

**Pull request closed without merging**